packages/nextra-theme-docs/css/styles.css

```
@@ -8,6 +8,7 @@
 @import 'nextra/styles/cards.css';
 @import './hamburger.css';
 @import './typesetting-article.css';
+@import 'nextra/styles/rmiz.css';
```

```suggestion
@import 'nextra/styles/react-medium-image-zoom.css';
```
let's be more precise